bridge/github/main.ml

```diff
@@ -79,6 +79,16 @@ let ( >>~ ) x f =
   | Error e -> Lwt.fail_with @@ Fmt.strf "%a" DK.pp_error e
   | Ok t    -> f t
 
+let () =
+  Lwt.async_exception_hook := (function
+      | Unix.Unix_error (error, func, arg) ->
+        Logs.warn (fun m ->
+            m  "Client connection error %s: %s(%S)"
```

Comment:
This seems like a potentially misleading error. There's no particular reason to think this is related to client connections. Might make more sense to define a wrapper around `Lwt.async` that takes a context message to print.